### PR TITLE
Rustbridge with spacekookie

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

**Comment by Hoverbear** [comment]
Yay!!! @spacekookie! =D

File: _includes/session_data.html [hunk 1]
```diff
@@ -5,7 +5,11 @@
 <div class="popout">
   <section>
       <h1>{{talk.title}}</h1>
-      <h2>by {% for speaker in talk.speakers %}{{ site.data.speakers[speaker].name }} {% endfor %}</h2>
+      <h2>by
+        {% for speaker in talk.speakers %}
+        {{ site.data.speakers[speaker].name }}
+        {% unless forloop.last %}and{% endunless %}
```
Comment (Hoverbear) [inline _includes/session_data.html:11]: This will have a problem if we have three speakers `x and y and z` but it doesn't matter right now.

**Review by Hoverbear — APPROVED** [review]

File: _includes/session_data.html (hunk 1 above)
Comment (badboy) [reply _includes/session_data.html:11]: I know, we don't have that in Paris so I didn't care to fix it yet.

**Comment by spacekookie** [comment]
Nice :tada:

**Review by flaki — APPROVED** [review]

**Comment by flaki** [comment]
Excellent, thank you Jan-Erik! I've merged and will tweet next week!

**Comment by flaki** [comment]
:bird: https://twitter.com/RustFest/status/993509146803818496

**Pull request merged**